frontend/javascript/components/docupload/attachment-manager.vue

```diff
@@ -0,0 +1,117 @@
+<script setup>
+
+import DjangoSlot from '../../lib/django-slot.vue'
+import FileUploader from '../upload/file-uploader.vue'
+import ImagesConverter from '../docupload/images-converter.vue'
+import AttachmentsTable from './attachments-table.vue'
+// TODO linter wrong? the two above are just fine...
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import OnlineHelp from '../online-help.vue'
+import BsToast from '../bs-toast.vue'
+import { useI18n } from '../../lib/i18n'
+import { useAttachments } from './lib/attachments'
+
+import { onMounted, nextTick, provide, ref, watch } from 'vue'
+
+const props = defineProps(['config', 'message'])
+
+const { i18n } = useI18n(props.config)
+provide('i18n', i18n)
+provide('config', props.config)
+
+const { attachments, addFromUppy, refresh } = useAttachments({
+  urls: {
+    ...props.config.url,
+    getAttachment: props.config.url.getAttachment.replace('/0/', '/') +
+      '?belongs_to=' +
+      props.message.id,
+  },
+  csrfToken: document.querySelector('[name=csrfmiddlewaretoken]').value,
+  i18n,
+})
+
+onMounted(() => refresh())
+
+const imagesConverterContainer = ref()
+
+watch(
+  () => attachments.images.length,
+  (newValue) => {
+    if (newValue > 0) {
+      nextTick().then(() => {
+        imagesConverterContainer.value?.scrollIntoView({ behavior: 'smooth', block: 'nearest' })
+      })
+    }
+  }
```

Comment:
Bit of a preference thing, but I'd prefer:


```suggestion
  async (newValue) => {
    if (newValue > 0) {
      await nextTick()
      imagesConverterContainer.value?.scrollIntoView({ behavior: 'smooth', block: 'nearest' })
    }
  }
```

Less nesting usually means more readable imho

Reply:
[3f7bc63](https://github.com/okfde/froide/pull/960/commits/3f7bc632ec42b57655ae7b5f9a9a3df6932487b7)